### ISSUE TITLE
Cropped information

### ISSUE THREAD

**Issue by Rowin63** [osmandapp/OsmAnd#15924]
### Description
Info in data fieldiscropped (time, distance, ...)

### How to reproduce?
see screenshot

### Your Environment
OsmAnd Version: 4.3.2.+
Android/iOS version: 12
Device model: Samsung Tab Active 3

![2022-12-09 Cropped Information](https://user-images.githubusercontent.com/95540936/206749933-4867b5e3-c090-490c-9f0d-504c013a9c8b.JPG)

**Comment by sonora** [comment]
No, font size was not changed. It's a known issue introduced yesterday which limited the widget wdth, see https://github.com/osmandapp/OsmAnd/issues/15074#issuecomment-1343044884

**Comment by mdykierek** [comment]
I installed new version today, so fixed or not it is not on publicly available build (download/releases either). I don't see #15924 being scheduled too. So the question is, when it will become available?

**Comment by vshcherb** [comment]
4.3.3 this week

**Comment by sonora** [comment]
@vshcherb Quick question: When we publish the non- beta release, will we follow common practice and _continue to increment_  the version numbering (to e.g. 4.3.4) right?

Because last time we "_restarted_" after beta 4.2.2 to non-beta 4.2.0, and that caused some unnecessary confusion supporting users.. Thx!

**Comment by Rowin63** [comment]
Hello, one additional question: is this also part of this issue? (Tab Active 3 / And 12 / Osmand+ 4.3.2)
Thanks!

![Offset](https://user-images.githubusercontent.com/95540936/207330000-9af1111e-2d87-42dd-94a1-36350cfd72be.JPG)
